components/admin_console/team_channel_settings/channel/details/channel_moderation.tsx

```diff
@@ -35,15 +35,15 @@ const formattedMessages: any = defineMessages({
         },
         disabledGuests: {
             id: t('admin.channel_settings.channel_moderation.createPosts.disabledGuest'),
-            defaultMessage: 'Create posts for guests are disabled in [{scheme_name}](../permissions/{scheme_link}).',
+            defaultMessage: 'Create posts for guests are disabled in [{scheme_name}](../permissions/{scheme_link}?rowIdFromQuery=create_post).',
```

Comment:
Another concern but could this block `?rowIdFromQuery=create_post` be a value that you pass in instead of something thats hard coded in the translation? Since its not going to be changing between translations it would be better to pass it as a value instead of requiring each translation to have the query parameter set properly

Reply:
Great idea! Done 😄 
![dancing](https://media.giphy.com/media/b5LTssxCLpvVe/giphy.gif)